CONTRIBUTING.md

```diff
@@ -93,6 +93,56 @@ The Continuous Integration (CI) runs automatically when a pull request is opened
 - Compilation: the code will be compiled using [Github Actions](https://github.com/features/actions) under Linux, Windows and MacOS, and the unit tests will be run;
 - SonarCloud will report the code smells, duplications and the code coverage. You have to fix all the relevant code smells and add unit tests to reach the required barrier.
 
+## Committer rights on one or several repositories
+
+### Committer rights
+
+A Committer is a developer who has the right to merge on protected branches on a given repository.
+A Committer may have rights on one or several repositories according to their expertise.
+
+A Committer is able to vote when there is someone asking for Committer's rights on a repository.
+
+A Committer is able to vote when there is a candidate for a TSC member seat.
+
+### Committer responsibilities
+
+**Being a Committer is a huge responsibility.**
+By asking for Committer's rights, the developer must keep in mind the following points:
+- Merge with care: code quality must be considered as much as possible and if technical debt is created, it should be justified and tracked (via an issue for example);
+- Participate in the Technical Steering Committee: it is expected for Committers to assist to TSC and get involved in their field of expertise;
+- Be active in the community: Committers should be reachable on Slack in order to answer to the questions of the community that are in their field of expertise;
+- Use their right to vote for new TSC members or new Committers: Committers are expected to answer to polls if they are able to do so (i.e. if  they are not away from the office).
```

Comment:
"Expected" is not a strict obligation, so no need to be more precise. Besides, some people only contribute when they are away from the office in open-source! :smiley: 
```suggestion
- Use their right to vote for new TSC members or new Committers: Committers are expected to answer to polls.
```